pkg/descriptions.json

```diff
@@ -116,6 +116,11 @@
     "CommonTypes": []
   },
   {
+    "Name": "Microsoft.Private.CoreFx.UAP",
+    "Description": "Package used to represent the portions of NETCoreApp that come from CoreFx for UAP.",
```

Comment:
This should not be talking about NetCoreApp, right? I think it should say:

`Package used to represent the portions of UAP that come from CoreFx.`